sdk/appcenter/src/main/java/com/microsoft/appcenter/persistence/DatabasePersistence.java

```diff
@@ -55,6 +58,22 @@
      */
     private static final ContentValues SCHEMA = getContentValues("", "");
 
+    /**
+     * Size limit for a database row log payload.
+     * A separate file is used if payload is larger.
+     */
+    private static final int PAYLOAD_MAX_SIZE = (int) (1.9 * 1024 * 1024);
```

Comment:
I know it's possible, but is it efficient to have 2 megabyte rows in the database?

Reply:
Also can you add to the comment that the size is in megabytes?

Reply:
I don't see any reason to lower the limit, plus Apple implementation will keep using SQLite for any size. Will update the comment.

Reply:
I was just curious because if we had rows that contained data in excess of 2 megabytes, certain queries could take much longer. but I don't know if that's true or affects the sdk.

Reply:
Will also fix size check based on bytes and not String chars.